jetty-maven-plugin/src/it/jetty-deploy-war-mojo-it/src/test/java/org/eclipse/jetty/its/jetty_run_war_mojo_it/TestHelloServlet.java

```diff
@@ -0,0 +1,93 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2018 Mort Bay Consulting Pty. Ltd.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+
+package org.eclipse.jetty.its.jetty_run_war_mojo_it;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.LineNumberReader;
+
+import org.eclipse.jetty.client.HttpClient;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ *
+ */
+public class TestHelloServlet
```

Comment:
Can you change the name of this class? It looks like it is a Servlet (like the other tests use), but its actually not - I was confused for a while.

Reply:
done